### ISSUE TITLE
4.1.2に実装例を追加する

### ISSUE THREAD

**Issue by tokimari** [openameba/a11y-guidelines#421]
### 概要 / Overview

#417 で [4.1.2 カスタムコントロールの操作性を担保する - Ameba Accessibility Guidelines](https://a11y-guidelines.ameba.design/4/1/2) の内容を大幅に書き換えました。その時実装例に掲載したコンポーネントがちょっとマニアックなものだったので、イメージが沸きづらいかも？という難点がありました。
なので、その後Spindle UIに追加された [NavigationTab/UnderlineTab](https://ameba-spindle.web.app/?path=/docs/navigationtab-underlinetab--variant-fixed) （いわゆるタブコンポーネント）の例に差し替えたいです。

#### やること

- [ ] https://a11y-guidelines.ameba.design/4/1/2 を読んでざっくり理解する
- [ ] https://waic.jp/translations/WCAG21/Understanding/name-role-value.html を読んでWCAG 2.1の4.1.2について理解する
- [ ] [具体例](https://a11y-guidelines.ameba.design/4/1/2/#segmentedcontrol)の内容をUnderlineTabに置き換える
- [ ] [テスト・チェック方法 Web](https://a11y-guidelines.ameba.design/4/1/2/#web)の内容をUnderlineTabに置き換える

### 関連リンク / Related link

https://github.com/openameba/a11y-guidelines/pull/417#discussion_r1354121730

**Comment by tokimari** [comment]
@yasuda-shin @itsminadesu  これ、追加コンポーネントがなんだったか忘れてしまったんですがなんでしたっけ・・w（というか公開されましたっけ）
ref: https://github.com/openameba/a11y-guidelines/pull/417#issuecomment-1754701125
ref: https://github.com/openameba/a11y-guidelines/pull/417#discussion_r1351981464

**Comment by itsminadesu** [comment]
上記のやり取りより後にリリースされたUIコンポーネントって[NavigationTab/UnderlineTab](https://ameba-spindle.web.app/?path=/docs/navigationtab-underlinetab--variant-fixed)しかなかった記憶なので、それですかね...？ 🤔

**Comment by yasuda-shin** [comment]
追加するのはSegmented Controlで対応終わって単純にこのissueがcloseされてないという気がしてます。

**Comment by tokimari** [comment]
そうだそれ！UnderlineTabですね！！！

**Comment by tokimari** [comment]
@newt239 こちらよろしくお願いします！不明点あればこちらででもSlackででもご相談ください〜！ 🙏
